### PR TITLE
- Miscellaneous fixes that I've lumped together. (Targeted to dev)

### PR DESCRIPTION
**Pull request by mjr4077au**
Most of these are bad variable accesses while running under `Set-StrictMode -Version Latest`. I'd implore you to consider enabling this mode within your module by default to allow stricter checks against undefined behaviour.

- `$configToastDisable` should be a bool and not a string.
- Ensure `$HKULanguages` is defined before attempting to access the variable.
  * Such setups breaks PowerShell when running strict compliance mode, such as `Set-StrictMode -Version Latest`.
- Set `$DisableLogging` to `$true` in `Write-Log` if `$LogFileName` is null/hasn't been set up yet.
  * Was flagging either under PowerShell 7, or when strict mode was on as the call to `Out-File` later on was firing before the `$logFile` global was initialised.
- Remove dead code relating to `$formCloseApps`.
  * Variable seems to be a remnant from a previous setup.
  * The remains of this was causing the script to throw under strict compliance mode.
- Repair `$script:notifyIcon` tests.
  * This was throwing under strict compliance mode as the variable is not guaranteed to be there. This check is safer under such situations.
- Ensure `$stdOut` and `$stdErr` are always available on the stack in `Execute-Process`.
  * These variables were only being defined deep within a branch, then the resulting attempt to access was against variables that possibly didn't exist.
- Leverage pre-existing `$culture` global inside `Invoke-RegisterOrUnregisterDLL`.
- Repair bad `$IsMspFile` setup that caused the upcoming branch to fail in strict compliance mode.

 
Before submitting this Pull Request, I made sure:

- [X] I tested the toolkit with my changes and made sure it doesn't break other code.

- [X] I updated the documentation with the changes I made.

- [X] The code I changed has comments with explanation.

- [X] The encoding of the file wasn't changed. It is still UTF8 with BOM.

### CODE REVIEW

File: Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1 [hunk 1]
```diff
@@ -1694,11 +1698,6 @@ https://psappdeploytoolkit.com
     ## Get the name of this function
     [String]${CmdletName} = $PSCmdlet.MyInvocation.MyCommand.Name
 
-    ## Stop the Close Program Dialog if running
-    If ($formCloseApps) {
-        $formCloseApps.Close
```
Comment (sintaxasn) [inline Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:1699]: Did you have a reason for removing this ?

File: Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1 [hunk 2]
```diff
@@ -1791,7 +1790,7 @@ https://psappdeploytoolkit.com
         New-ZipFile -DestinationArchiveDirectoryPath $configToolkitLogDir -DestinationArchiveFileName $DestinationArchiveFileName -SourceDirectory $logTempFolder -RemoveSourceAfterArchiving
     }
 
-    If ($script:notifyIcon) {
+    If (Test-Path -LiteralPath 'variable:notifyIcon') {
```
Comment (sintaxasn) [inline Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:1793]: What's the purpose behind this change?

File: Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1 [hunk 3]
```diff
@@ -10641,7 +10641,7 @@ https://psappdeploytoolkit.com
             Return
         }
         ## Dispose of previous balloon
-        If ($script:notifyIcon) {
+        If (Test-Path -LiteralPath 'variable:notifyIcon') {
```
Comment (sintaxasn) [inline Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:10644]: Same question here

File: Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1 (hunk 1 above)
Comment (mjr4077au) [reply Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:1699]: It might be best to read the PR descriptions for some of these. I enabled strict compliance mode (`Set-StrictMode -Version Latest`) and started to comb through the errors. This variable is not defined anywhere at all, therefore it's an access on an undefined variable and was causing a throw. It looks like something that's been left behind from a previous setup.

File: Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1 (hunk 2 above)
Comment (mjr4077au) [reply Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:1793]: Same here, it's an attempt to access a variable that doesn't always exist depending on whether the toolkit is running silent or not. Under PowerShell's loosey-goosey default mode, it slips through but in strict compliance mode (`Set-StrictMode -Version Latest`), it causes a throw under the right conditions.

File: Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1 (hunk 3 above)
Comment (mjr4077au) [reply Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:10644]: Same answer here as well.

Comment (sintaxasn) [reply Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:10644]: I think it'd be useful to have a chat about same of these. If you're on Discord, maybe hit me up there?

Comment (mjr4077au) [reply Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:10644]: Do you have a link at all? The one on the website seemed dead

Comment (sintaxasn) [reply Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1:10644]: 🤦🏻https://discord.gg/winadmins
